marimbabot_hardware/src/servo_interface.cpp

```diff
@@ -98,7 +98,7 @@ bool ServoInterface::try_open_serial_port() {
             this->top_limit = atoi(top_limit_str.c_str());
             this->bottom_limit = atoi(bottom_limit_str.c_str());
             this->resolution = atoi(resolution_str.c_str());
-            this->radian_limit = (2 * M_PI) * ((this->top_limit - this->bottom_limit) / ((double) this->resolution));
+            this->radian_limit = M_PI * ((this->top_limit - this->bottom_limit) / ((double) this->resolution));
```

Comment:
I am not sure about this formula. why is there pi in there? What is the unit of the limits?

Reply:
not related to this modification but, catkin throws error due to arduino's serial library in servo_interface.cpp and somehow it is merged to main branch. Did you also modify cmake file after you add serial library ?

Reply:
@berkgungor are you sure you reran rosdep? serial is specified in the package.xml but you have to install it with rosdep as it is a new dependency

Reply:
@berkgungor Is it solved? I guess it was just merged locally

Reply:
serial library requires manual installation on locals, catkin build throws error after rosdep update. Problem is solved after manual installation but would be nice if noted on read.me or somewhere.